code/game/objects/items/ego_weapons/teth.dm

```diff
@@ -187,3 +187,15 @@
 	attack_verb_continuous = list("jabs")
 	attack_verb_simple = list("jabs")
 	hitsound = 'sound/weapons/slashmiss.ogg'
+
+/obj/item/ego_weapon/mini/bean
+	name = "magic bean"
+	desc = "...We may never find out what lies at the top, but perhaps those who made it are doing well up there."
+	icon_state = "bean"
+	force = 19
+	attack_speed = 1
```

Comment:
Attack speed is considered 1 if not changed.

Reply:
That is the teth EGO

Reply:
this can be fixed